pkg/kubelet/certificate/certificate_manager.go

```diff
@@ -156,6 +159,12 @@ func (m *manager) Current() *tls.Certificate {
 	return m.cert
 }
 
+// CertificateSigningRequestClient sets the client interface that is used for
+// signing new certificates generated as part of rotation.
+func (m *manager) CertificateSigningRequestClient(certSigningRequestClient certificatesclient.CertificateSigningRequestInterface) {
```

Comment:
Is this only used by the test? If so, can this just be on the internal type? Alternatively, create a "Test" or "Internal" interface that adds this method.